### PR TITLE
Add --help command for the runtests.sh script

### PR DESCRIPTION
**Pull request by m4sk1n**
Simple, but I don't think there is need for anything more…

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----

### CODE REVIEW

File: runtests.sh [hunk 1]
```diff
@@ -20,11 +20,15 @@ set -e
 export PATH=$(pwd)/bin:$PATH
 export PYTHONPATH=$(pwd)${PYTHONPATH:+:$PYTHONPATH}
 
+printhelp(){
+    echo "Usage: ./runtests.sh unit|snaps|<snapcraft-suite>"
```
Comment (sergiusens) [inline runtests.sh]: I know this is not on you, but mind updating this to be 
```bash
    echo "Usage: "
    echo "    ./runtests.sh static"
    echo "    ./runtests.sh snapcraft/tests/unit"
    echo "    ./runtests.sh snapcraft/tests/integration[/<test-suite>]"
    echo "    ./runtests.sh snaps"
    echo ""
    echo "<test-suite> can be: $(ls snapcraft/tests/integration| grep '^[a-z].*' | tr '\n' ' ')"
```

File: runtests.sh [hunk 2]
```diff
@@ -20,11 +20,15 @@ set -e
...
+    echo "<snapcraft-suite> can be snapcraft/tests, or any of its subdirectories."
```
Comment (sergiusens) [inline runtests.sh]: this line should go away after the edit above

**Comment by m4sk1n** [comment]
@sergiusens done :)

**Comment by kyrofa** [comment]
Could you please add a reference to the bug being fixed, here?

**Comment by kyrofa** [comment]
Ah, found it in GCI: LP: [#1702716](https://bugs.launchpad.net/snapcraft/+bug/1702716).

**Comment by sergiusens** [comment]
Great work!

**Pull request merged**